src/runtime/node/cluster/index.ts

```diff
@@ -0,0 +1,51 @@
+import noop from "../../mock/noop";
+import mock from "../../mock/proxy";
+import type cluster from "node:cluster";
+import type { Cluster, Worker as _Worker } from "node:cluster";
+import { EventEmitter } from "../events";
+
+/**
+ * Inspired by the original Node.js code in:
+ * https://github.com/nodejs/node/blob/main/lib/internal/cluster/primary.js
+ */
+
+// A mapped type used internally to allow assigning to readonly fields like `isPrimary`
+type MutableCluster = {
+  -readonly [key in keyof Cluster]: Cluster[key];
+} & {
+  Worker: typeof _Worker;
+};
+
+// eslint-disable-next-line unicorn/prefer-event-target
+const _cluster = new EventEmitter() as MutableCluster;
+
```

Comment:
did you intentionally skip:

_events
_eventsCount
_maxListeners

I think we should add them as they are part of the public api even though they are `_`-prefixed.

Reply:
We get `_events` and `_maxListeners` for free now that we're extending `EventEmitter` (defined [here](https://github.com/unjs/unenv/blob/main/src/runtime/node/events/_events.ts#L37-L40)). They aren't defined on the `Cluster` type though. I can probably add `_eventsCount` there as well if that's cool with @pi0.

Reply:
I see! but don't you still need add `export const _events`, etc. to support named imports?

Reply:
Yeah good point...

Reply:
ok fixed